docs/spec/State-Machine-Behavior-Elements/State-Transition-Specifiers.adoc

```diff
@@ -40,13 +45,13 @@ When making a self transition, the state machine runs the code associated with
 exiting and then re-entering the state.
 
 . Second form of the syntax specifies an
-*internal transition*, i.e., an action to take while remaining
+*internal transition*, i.e., a list of actions to take while remaining
 in the same state.
 When making an internal transition, the exit and re-entry code is not run.
-The identifier after the keyword `do` must
+The list of actions specified in the _do-expression) must
```

Comment:
No italics or underscore in "do expression."

Reply:
Fixed